src/main.rs

```diff
@@ -53,24 +63,45 @@ fn main() -> Result<()> {
     };
 
     if let Some(Commands::Stats { all }) = cli.command {
+        if cli.query.is_some() {
+            // todo implement --query for stats
+            error!("Sorry, '--query' is not implemented for stats yet.");
```

Comment:
Das rennt einfach weiter nachdem der error geloggt wurde, soll das so? Meine Intuition wäre abort für invalid args,

```suggestion
            bail!("Sorry, '--query' is not implemented for stats yet.");
```

Reply:
👍